import java.util.Scanner;
class UsernameException extends Exception
{
    public UsernameException(String msg) {
        super(msg);
    }
}
class PasswordException extends Exception
{
    public PasswordException(String msg) {
        super(msg);
    }
}
class DemoExcp
{
    public static void main(String args[])
    {
        String name="Arjun";
        String pass="Arjun01";
        Scanner s= new Scanner(System.in);
        System.out.print("Enter username :: ");
        String user=s.nextLine();
        System.out.print("Enter password :: ");
        String pwd=s.nextLine();
        try
        {
            if(!user.equals(name))
                throw new UsernameException("Username incorrect");
            else if(!pwd.equals(pass))
                throw new PasswordException("Password incorrect");
            else
                System.out.println("Login Successful !!!");
        }
        catch(UsernameException u)
        {
            u.printStackTrace();
        }
        catch(PasswordException p)
        {
            p.printStackTrace();
        }
    }
}




**OUTPUT**

"C:\Program Files\Java\jdk-21\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.3.3\lib\idea_rt.jar=65319:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.3.3\bin" -Dfile.encoding=UTF-8 -Dsun.stdout.encoding=UTF-8 -Dsun.stderr.encoding=UTF-8 -classpath C:\Users\mlm\IdeaProjects\untitled1\out\production\untitled1 DemoExcp
Enter username :: Arjun
Enter password :: Arjun01
Login Successful !!!

Process finished with exit code 0

